tests/unit/model/column_test.py

```diff
@@ -97,6 +103,28 @@ def test_create_when_parent_is_table():
     assert sut.table_name == TABLE_NAME
 
 
+def test_create_when_parent_is_snowflake_dynamic_table():
+    sut = Column.create(
+        name=COLUMN_NAME,
+        parent_qualified_name=TABLE_QUALIFIED_NAME,
+        parent_type=SnowflakeDynamicTable,
+        order=1,
+    )
+
+    assert sut.name == COLUMN_NAME
+    assert sut.qualified_name == TABLE_COLUMN_QUALIFIED_NAME
+    assert sut.connector_name == CONNECTOR_TYPE
+    assert sut.schema_name == SCHEMA_NAME
+    assert sut.schema_qualified_name == SCHEMA_QUALIFIED_NAME
+    assert sut.database_name == DATABASE_NAME
+    assert sut.database_qualified_name == DATABASE_QUALIFIED_NAME
+    assert sut.connection_qualified_name == CONNECTION_QUALIFIED_NAME
+    assert sut.order == 1
+    assert sut.table_qualified_name == TABLE_QUALIFIED_NAME
+    assert sut.table.qualified_name == TABLE_QUALIFIED_NAME
```

Comment:
Same test assertion is listed twice here ^

Reply:
Fixed.